modules/wechat_qrcode/include/opencv2/wechat_qrcode.hpp

```diff
@@ -0,0 +1,89 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Tencent is pleased to support the open source community by making WeChat QRCode available.
+// Copyright (C) 2020 THL A29 Limited, a Tencent company. All rights reserved.
+
+#ifndef __OPENCV_WECHAT_QRCODE_HPP__
+#define __OPENCV_WECHAT_QRCODE_HPP__
+#include "opencv2/core.hpp"
+#include "opencv2/wechat_qrcode/align.hpp"
+#include "opencv2/wechat_qrcode/ssd_detector.hpp"
+#include "opencv2/wechat_qrcode/super_scale.hpp"
+
+/** @defgroup wechat_qrcode WeChat QR code detector for detecting and parsing QR code.
+*/
+namespace cv {
+namespace wechat_qrcode {
+using std::string;
```

Comment:
please, do not define such global aliases for std types in public headers. use explicit `std::`. Such aliases can be moved to precomp.hpp